.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+### Instructions
+
+- Use a **concise** yet **descriptive** title;
```

Comment:
I think this is what we're discussing below re: pointing ppl to guidelines for writing commit messages.